addons/event/views/event_event_views.xml

```diff
@@ -232,9 +232,10 @@
                                         <div class="o_kanban_record_title o_text_overflow" t-att-title="record.name.value">
                                             <field name="name"/>
                                         </div>
-                                        <div t-if="record.address_id.value" class="d-flex ps-1">
+                                        <div class="d-flex ps-1">
                                             <i class="fa fa-map-marker mt-1 me-2 text-center ps-1" title="Location"/>
-                                            <span t-esc="record.address_id.value" class="ms-1"/>
+                                            <span t-if="record.address_id.value" t-out="record.address_id.value" class="ms-1"/>
+                                            <span t-else="" t-out="'Online event'" class="ms-1"/>
```

Comment:
Please re-read my previous comment, you cannot use t-esc or t-out for translated terms as they will not appear in the POT file, I left the correct suggestion in my previous review.

Reply:
Hello @awa-odoo 
My bad I misunderstood that one
Done with the changes. Can you please have a look?

Thanks!!